examples/lltest.py

```diff
@@ -23,7 +22,6 @@
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 '''
 
-from __future__ import division, print_function, absolute_import
```

Comment:
this and a lot of the other removed code is for python 2.x, which we do not test for any more. nor our setup.py declares support for 2.x.

but, the readme still states one could use llfuse if one is stuck with python2.
so, if we remove this code, the readme needs an update also.

if there is any user still on 2.7, they can use an older release that still has this code.
this should be added to the readme: if you are stuck on python 2.x, use `llfuse<1.4.0`.